### ISSUE TITLE
parse charge error in parsePQR

### ISSUE THREAD

**Issue by jjgoings** [prody/ProDy#740]
Using ProDy 1.10.7 (commit [#48619e85294786](https://github.com/prody/ProDy/commit/48619e85294786c869547d3e69f737c6a7594bbb)) on MacOS 10.13.6, I get an error when using `parsePQR`.

I am trying to parse the Fasciculin-2 (1FSC) PQR from ABPS examples. The input PQR can be found here: [fas2.pqr](http://www.poissonboltzmann.org/apbs/examples/misc/fas2.pqr)

My input is
```
import prody as pd

mypqr = 'fas2.pqr'
parsed = pd.parsePQR(mypqr)
```

which returns the error

```
@> WARNING failed to parse charge at line 0
@> WARNING failed to parse charge at line 1
@> WARNING failed to parse charge at line 2
@> WARNING failed to parse charge at line 3
# ... rest of lines continue ...
@> WARNING failed to parse charge at line 901
@> WARNING failed to parse charge at line 902
@> WARNING failed to parse charge at line 903
@> WARNING failed to parse charge at line 904
@> WARNING failed to parse charge at line 905
Traceback (most recent call last):
  File "testprody.py", line 4, in <module>
    parsed = pd.parsePQR(mypqr)
  File "/anaconda/envs/mdaenv/lib/python2.7/site-packages/ProDy-1.10.7-py2.7-macosx-10.9-x86_64.egg/prody/proteins/pdbfile.py", line 360, in parsePQR
    subset=subset, altloc_torf=False, format='pqr')
  File "/anaconda/envs/mdaenv/lib/python2.7/site-packages/ProDy-1.10.7-py2.7-macosx-10.9-x86_64.egg/prody/proteins/pdbfile.py", line 594, in _parsePDBLines
    charges = np.concatenate((charges,
UnboundLocalError: local variable 'charges' referenced before assignment
```

**Comment by jamesmkrieger** [comment]
Thanks for reporting this. I have now fixed it (see #741). Please make this change in your code https://github.com/prody/ProDy/pull/741/commits/6deb24a45b53b1d83a10960b6789f365b2a22379

Sorry about that.

**Comment by jamesmkrieger** [comment]
This now merged so you should be able to update your prody with git pull to fix it. Please reopen this issue and let me know if you have any other problem with this.

Best wishes
James
